srv/_utils/process.py

```diff
@@ -1,4 +1,3 @@
-#
 # Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Not necessary, instead use https://github.com/Seagate/cortx-prvsnr/blob/99eef712cdab764f34ef8cd7f2622ee801fea037/api/python/provisioner/utils.py#L299